css/amp.css

```diff
@@ -684,6 +684,37 @@ i-amphtml-video-mask, i-amp-video-mask {
   100% {transform: translateY(0);}
 }
 
+
+/**
+ * Video Docking
+ */
+
+.i-amphtml-dockable-video {
+  padding: 0px;
+  margin:0px;
+  background: darkgray;
+}
+
+.i-amphtml-dockable-video > video.i-amphtml-dockable-video-minimizing,
+.i-amphtml-dockable-video > iframe.i-amphtml-dockable-video-minimizing {
+  position: fixed;
+  box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.3);
+  border-radius: 6px;
```

Comment:
not a big fan of the rounded corners, can we keep it square initially and reevaluate with @spacedino later?